axlearn/common/inference_test.py

```diff
@@ -20,6 +20,7 @@
 from absl.testing import absltest, parameterized
 from jax.experimental import mesh_utils
 from jax.experimental.pjit import pjit
+import pytest
```

Comment:
is this needed?

Reply:
https://github.com/ptoulme-aws/axlearn/blob/52d0991ed0224f034ccf82b088d7e39b112cd302/axlearn/common/inference_test.py#L325